uber/models.py

```diff
@@ -642,6 +642,8 @@ def merch(self):
         merch = self.donation_swag
```

Comment:
The shirt size is displayed as part of the "Choose a shirt size" selection drop-down on the merch page, so that's covered.